packages/core/src/progress-circle/progress-circle.stories.ts

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import '@clr/core/icon/register.js';
+import '@clr/core/progress-circle/register.js';
+import { ClarityIcons, homeIcon, nodesIcon, stopIcon, warningStandardIcon } from '@clr/core/icon';
+import { getElementStorybookArgTypes, spreadProps, getElementStorybookArgs } from '@clr/core/internal';
+import { html } from 'lit-html';
+import customElements from '../../dist/core/custom-elements.json';
+
+ClarityIcons.addIcons(homeIcon, nodesIcon, stopIcon, warningStandardIcon);
+
+export default {
+  title: 'Components (Preview)/Circular Progress/Stories',
+  component: 'cds-progress-circle',
+  argTypes: getElementStorybookArgTypes('cds-progress-circle', customElements),
+  parameters: {
+    options: { showPanel: true },
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/v2mkhzKQdhECXOx8BElgdA/Clarity-UI-Library---light-2.2.0?node-id=51%3A673',
+    },
+  },
+};
+
+export const API = (args: any) => {
+  return html` <cds-progress-circle ...="${spreadProps(getElementStorybookArgs(args))}"></cds-progress-circle> `;
+};
+
+export const spinner = () => {
+  return html`
+    <cds-progress-circle
+      size="xl"
+      aria-label="Demonstration of indeterminate progress circle spinning to show work is in progress"
+    ></cds-progress-circle>
+  `;
+};
+
+export const status = () => {
+  const labelText = 'Demonstration of circular progress ';
+  return html`
+    <div cds-layout="horizontal gap:sm">
+      <cds-progress-circle
+        size="xl"
+        value="0"
+        role="progressbar"
+        aria-valuenow="0"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        value="12"
+        role="progressbar"
+        aria-valuenow="12"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        value="100"
+        role="progressbar"
+        aria-valuenow="100"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        status="info"
+        value="48"
+        role="progressbar"
+        aria-valuenow="48"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        status="success"
+        value="72"
+        role="progressbar"
+        aria-valuenow="72"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        status="warning"
+        value="36"
+        role="progressbar"
+        aria-valuenow="36"
+        aria-valuemin="0"
+        aria-valuemax="100"
```

Comment:
Following the lead of other core components, I have left much of the aria work on the product side. All of the examples are based on level access progress bar examples.